Networking/Networking/ApplicationPassword/RequestProcessor.swift

```diff
@@ -3,8 +3,8 @@ import Foundation
 
 /// Authenticates and retries requests
 ///
-final class RequestProcessor {
-    private var requestsToRetry = [RequestRetryCompletion]()
+final class RequestProcessor: RequestInterceptor {
```

Comment:
Note: [Alamofire.RequestInterceptor](https://github.com/Alamofire/Alamofire/blob/5.9.0/Source/Features/RequestInterceptor.swift#L118) is `RequestAdapter` & `RequestRetrier`, which `RequestProcessor` already conforms to.